scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -620,10 +618,17 @@ public static BuildTargetResult CrossgenSharedFx(BuildTargetContext c, string pa
                 // in CompileTargets and the one in the shared library project.json match and are updated in lock step, but long term
                 // we need to be able to look at the project.lock.json file and figure out what version of Microsoft.NETCore.Runtime.CoreCLR
                 // was used, and then select that version.
-                ExecSilent(Crossgen.GetCrossgenPathForVersion(CompileTargets.CoreCLRVersion),
+                ExecSilent(Crossgen.GetCrossgenPathForVersion(CoreCLRVersion),
                     "-readytorun", "-in", file, "-out", tempPathName, "-platform_assemblies_paths", pathToAssemblies);
 
-                File.Delete(file);
+                for (int i = 0; i < 10 && File.Exists(file); i++)
+                {
+                    File.Delete(file);
```

Comment:
No, though I'm probably just going to revert it since I don't think it's actually doing anything to help :)